Limelight/Database/TemporarySettings.m

```diff
@@ -57,6 +57,10 @@ - (id) initFromSettings:(Settings*)settings {
             self.width = [NSNumber numberWithInteger:1920];
             break;
         case 2:
+            self.height = [NSNumber numberWithInteger:1440];
+            self.width = [NSNumber numberWithInteger:2560];
+            break;
+        case 3:
```

Comment:
It's not safe to use 2 for 1440p, because 2 is what is currently used for 4K. You should use 3 for 1440p instead.